common/reviews/api/rush-lib.api.md

```diff
@@ -430,13 +494,19 @@ export interface IOperationRunner {
 
 // @beta
 export interface IOperationRunnerContext {
+    readonly changedProjectsOnly: boolean;
     collatedWriter: CollatedWriter;
+    readonly consumers: Set<IOperationRunnerContext>;
     debugMode: boolean;
+    error?: Error;
     // @internal
     _operationMetadataManager?: _OperationMetadataManager;
     quietMode: boolean;
+    readonly runner: IOperationRunner;
+    status: OperationStatus;
     stdioSummarizer: StdioSummarizer;
-    stopwatch: IStopwatchResult;
+    // Warning: (ae-forgotten-export) The symbol "Stopwatch" needs to be exported by the entry point index.d.ts
```

Comment:
I have decided to revert it back to using `IStopwatchResult` and temporarily assert `context.stopwatch` to the `StopWatch` type. This is necessary because `CacheableOperationPlugin` requires calling the `reset` function to revive the stopwatch instance.